cmd/infracost/diff.go

```diff
@@ -45,14 +45,14 @@ func diffCmd(ctx *config.RunContext) *cobra.Command {
 				return err
 			}
 
-			ctx.Config.Format = "diff"
-
 			return runMain(cmd, ctx)
 		},
 	}
 
 	addRunFlags(cmd)
 
+	cmd.Flags().String("compare-to", "", "Path to Infracost JSON file to compare against, cannot be used with table and html formats")
```

Comment:
```suggestion
	cmd.Flags().String("compare-to", "", "Path to Infracost JSON file to compare against")
```
Am I right in saying the other part of the message is no longer needed?

Reply:
👍